index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">   
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+        <nav class ="navigation">
+            <ul >
+                <li class="awesome-link"><a>Awesome Books</a></li>
+                <li class="links" id="list-link"><a>List</a></li>
+                <li class="links" id="add-link"><a>Add new</a></li>
+                <li class="links" id="contact-link"><a>Contact</a></li>
+            </ul>
+        </nav>
+        <div id="date-time">Time</div>
+    </header>
+    <main>
+        <section class="box" id="list">
+            <h1>All awesome Books</h1>
+            <div class = "container">            
+            </div>       
```

Comment:
- [ ] [OPTIONAL] I suggest to consider adding the titles. This will help the application user be able to identify which column is for the author or book name. This will improve the application to be more user friendly ☺️.